style.css

```diff
@@ -0,0 +1,268 @@
+    height:100%;
+}
+nav {
+    margin: 0;
+    padding: 0;
+    background-color: #111111;
+    width: 100%;
+    position: sticky;
+    top: 0;
+    color: #dddddd;
+    font-family: 'Lato', sans-serif;
+    z-index:5;
+}
+#navwrapper {
+    display:-webkit-box;
+    display:-ms-flexbox;
+    display:flex;
+    width: 65%;
+    margin: 0 auto;
+    -webkit-box-pack: justify;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+}
+.navitem {
+    margin:0;
+    padding: 0 20px 0 20px;
+    -ms-flex: 0 1 auto
+}
+.navitem>h4 {
+    font-size:1.4rem;
+    margin: 10px 0 2px 0;
+}
+.navitem>p {
+    font-size:0.6rem;
+    margin: 2px 0 10px 0;
+}
+header {
+    margin: 0;
+    padding: 0;
+    background-color: #eeeeee;
+    height:auto;
+    overflow:hidden;
+}
+#headerwrapper {
+    width: 65%;
+    height:100%;
+    margin: 0 auto;
+    background-size: auto 100%;
+    background-image: url(headerbackground.png);
+    background-repeat: no-repeat;
+    background-position: right;
+    overflow:hidden;
+}
+#headertext {
+    width:100%;
+}
+header h1 {
+    font-family: 'Mukta', sans-serif;
+    margin: 0;
+    font-size:2.4rem;
+    padding: 40px 0 0 0;
+}
+header h2 {
+    font-family: 'Poppins', sans-serif;
+    font-size:1.8rem;
+    font-weight:600;
+    margin: 0;
+    padding:20px 0 10px 0;
+}
+header p {
+    font-family: 'Mukta', sans-serif;
+    font-size:0.8rem;
+    width:50%;
+    margin:0;
+    padding:0 0 10px 0;
+}
+#badges {
+    display: flex;
+    -ms-display:flex;
+    padding: 8px 0 20px 0;
+    font-size:2.6rem;
+}
+#showcase {
+    margin: 0;
+    padding: 0;
+    background-color: #bbbbbb;
+    width:100%;
+}
+#showcasewrapper {
+    width:65%;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-orient: horizontal;
+    -webkit-box-direction: normal;
+    -ms-flex-direction: row;
+    flex-direction: row;
+    margin: 0 auto;
+}
+#showcasetext {
+    width:40%
+}
+#showcasetext h2 {
+    font-family: 'Poppins', sans-serif;
+    font-size:1.8rem;
+    font-weight:600;
+    margin: 0;
+    padding:40px 0 10px 0;
+}
+#showcasetext h3 {
+    font-family: 'Poppins', sans-serif;
+    font-size:0.75rem;
+    width:90%;
+    margin:0;
+    padding:4px 0 4px 0;
+}
+#showcasetext p {
+    font-family: 'Mukta', sans-serif;
+    font-size:0.9rem;
+    width:90%;
+    margin:0;
+    padding:4px 0 40px 0;
+}
+#showcaseviewer {
+    width:60%;
+    height:266px;
+    border: 8px solid #111111;
+    border-radius: 6px;
+    margin:40px 0 40px 0;
+    background-image: url(projectexample.jpg);
+    background-size: 100% auto;
+    background-position: center;
+    background-repeat: no-repeat;
+}
+.showcasenav {
+    margin: 20% 0 25% 0;
+    background-color:rgba(102, 102, 102, 0.66);
+    color: #ffffff;
+    width: 40px;
+    height:30px;
+    border:0;
+}
+#showcaseleft {
+    float:left;
+}
+#showcaseright {
+    float:right;
+}
+#showcasebottom {
+    margin-top:224px;
```

Comment:
instead of using a fixed top margin. If you make the parent element position relative, and make this element position absolute, you can use bottom: 0; to place it at the bottom of the box without having a fixed margin size. making it more flexible.

Reply:
changed in cc1182b